packages/icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.322.0",
+  "version": "1.323.0",
```

Comment:
Fer it may be possible you should also update `pacjage-lock.json` file. Run npm i or yarn (not sure which one is used in this repo)

Reply:
Its a good call but I didnt see other PRs doing it. Perhaps we should check first if this is desired or not.

Reply:
In fact, it has been ignored. You can check the .gitignore file.